### ISSUE TITLE
When clicking FullName in detailpage, add path to clipboard

### ISSUE THREAD

**Issue by jipclaassens** [ObjectVision/GeoDMS#357]
When clicking FullName in detailpage, add path to clipboard

Now nothing happens, in Delphi you got a popup in which you could copy the path.

**Comment by MaartenHilferink** [comment]
You can select the fullname text with your mouse and then right-click->copy

**Comment by MaartenHilferink** [comment]
A user might not expect that clicking on a link changes the clipboard to what is after that link.

**Comment by eoudejans** [comment]
I agree to Maartens comment, at the moment the detail pages function as a browser, which allows for right-click >> copy and also mouse-drag-select and ctrl-c for copying relevant information.

**Comment by eoudejans** [comment]
Active item link in detail pages now has no use for the link. 
As suggestion: when the link is clicked, copy the text to clipboard using copy_link: code.

This should be accompanied by a temporarily visible hint that the active item is set to the user's clipboard.
